torch/_export/converter.py

```diff
@@ -773,3 +802,36 @@ def retrace_as_exported_program(
         ep.verifier().check(ep)
 
         return ep
+
+    def lift_tensor_constants_to_buffer(self):
+        # torchscript does not lift tensor constants to buffers. So we cannot access tensor
+        # constants during `graph_converter.convert()`.
+        attribute_map = {}
+
+        def get_attr(fqn: str):
+            name = fqn.split(".")
+            v = self.ts_model
+            for n in name:
+                v = getattr(v, n)
+            return v
+
+        for node in self.ts_graph.nodes():
+            if node.kind() == "prim::CreateObject":
+                output_name = node.output().debugName()
+                attribute_map[output_name] = ""
+            if node.kind() == "prim::GetAttr":
+                output_name = node.output().debugName()
+
+                attr_name = node.s("name")
+                input_name = node.input().debugName()
+
+                root_attr_name = attribute_map[input_name]
+                attr_fqn = (
+                    f"{root_attr_name}.{attr_name}" if root_attr_name else attr_name
+                )
+                value = get_attr(attr_fqn)
+
+                attribute_map[output_name] = attr_fqn
+                if isinstance(value, torch.Tensor):
+                    # Lift tensor constants to be a buffer
+                    self.name_to_buffer_map[attr_fqn] = value
```

Comment:
Should we only lift it if there's a setattr? If it's only a getattr could we just keep it as a tensor constant?